packages/block-library/src/button/edit.js

```diff
@@ -256,20 +252,25 @@ function ButtonEdit( props ) {
 					<LinkControl
 						value={ linkValue }
 						onChange={ ( {
-							url: newURL = '',
+							url: newURL,
 							opensInNewTab: newOpensInNewTab,
```

Comment:
Question: do we need these aliases? I'd like to contain all the logic for computing the new link value in the function rather than leaving anything outside if possible.

Reply:
Ah I see. I've seen this alias pattern used to ensure default values before, but I just realised the alias would need to have default values for that to be the case.

All good as is 👍

Reply:
Great! Thanks for all the dedicated reviews and suggestions